parts/kubernetesmasteraddons-tiller-deployment1.5.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
```

Comment:
Was missing a commit but the main difference is L9 using the alpha rbac API as beta isn't available in 1.5